design-tokens-manager/src/routes/Scaler/Results.jsx

```diff
@@ -0,0 +1,63 @@
+import { useEffect, useState } from 'react';
+import { Box, Text } from 'grommet';
+
+const roundToNearest = (value, nearest) => {
+  return Math.round(value / nearest) * nearest;
+};
+
+const createScale = (base, factor, steps, nearest) => {
+  // TO DO: Refactor this to be more efficient
+
+  const array = new Array(steps * 3).fill(0);
+  const result = array
+    .map((_, index) => {
+      return roundToNearest(Math.pow(factor, index) * base, nearest || base);
+    })
+    .reduce((acc, value) => {
+      if (!acc.includes(value)) {
+        acc.push(value);
+      }
+      return acc;
+    }, [])
+    .splice(0, steps);
+
+  // add values below the base to results, down to 0
+  for (let i = base; i >= 0; i--) {
```

Comment:
Did lint throw an issue here for the "--"? Feel like I usually get one and have to do `-= 1`

Reply:
No issue with linting.